graylog2-web-interface/src/views/components/sidebar/highlighting/HighlightingRule.jsx

```diff
@@ -1,14 +1,52 @@
 // @flow strict
 import * as React from 'react';
 import PropTypes from 'prop-types';
-
-import { ColorPickerPopover, Icon } from 'components/common';
+import styled, { type StyledComponent } from 'styled-components';
+import { type ThemeInterface } from 'theme';
 
 import { DEFAULT_CUSTOM_HIGHLIGHT_RANGE } from 'views/Constants';
 import Rule from 'views/logic/views/formatting/highlighting/HighlightingRule';
 import { HighlightingRulesActions } from 'views/stores/HighlightingRulesStore';
 
-import styles from './HighlightingRules.css';
+import { ColorPickerPopover, Icon } from 'components/common';
+import ColorPreview from './ColorPreview';
+
+export const HighlightingRuleGrid: StyledComponent<{}, void, HTMLDivElement> = styled.div`
+  display: grid;
+  display: -ms-grid;
+  margin-top: 5px;
+  grid-template-columns: max-content 1fr max-content;
+  -ms-grid-columns: max-content 1fr max-content;
+
+  > *:nth-child(1) {
+    grid-column: 1;
+    -ms-grid-column: 1;
+  }
+
+  > *:nth-child(2) {
+    grid-column: 2;
+    -ms-grid-column: 2;
+  }
+
+  > *:nth-child(3) {
+    grid-column: 3;
+    -ms-grid-column: 3;
+  }
+`;
```

Comment:
The above `nth-child` styling is only required to support IE11. I used `:nth-child` instead of defining the styling for each child because it allows a more flexible usage of the component.